### PR TITLE
PLUM-554: Do not allow None values in Config object

### PR DESCRIPTION
**Pull request by SukiCZ**
http://jira.teskalabs.int:8080/browse/PLUM-554

### CODE REVIEW

File: asab/config.py [hunk 1]
```diff
@@ -161,6 +161,10 @@ def __init__(self, config_section_name, config=None):
 
 			# Merge config defaults of each base class in the 'inheritance' way
 			for key, value in base_class.ConfigDefaults.items():
+
+				if value is None:
+					raise ValueError("%s value for key %s in %s" % (value, key, base_class))
```
Comment (ateska) [inline asab/config.py]: The message in the exception is not describing the nature of the problem.

Comment (SukiCZ) [reply asab/config.py]: @ateska Could you recommend me a better description then?

Comment (ateska) [reply asab/config.py]: "Don't use None in ConfigDefaults, found in {}:{}".format(section, key)

Comment (SukiCZ) [reply asab/config.py]: Example of formatted message

```
None value not allowed in ConfigDefaults. Found in connection:MySQLConnection1:connect_timeout
```

**Pull request merged**